### PR TITLE
Add assessment functionality

### PR DESCRIPTION
**Pull request by tim-macphail**
# Add assessment functionality
Also refactored the Assessment interface
https://github.com/techstartucalgary/lifeline/blob/b8ed354d7c14489b29fd8fb4dc52f45a10b8db88/client/src/logic/icsGen.ts#L3-L7

### CODE REVIEW

**Comment by tim-macphail** [comment]
Needs work

File: client/src/components/EditAssessment/EditAssessment.tsx [hunk 1]
```diff
@@ -15,16 +15,31 @@ const EditAssessment = ({
...
+    setWeight(Number(e.target.value.trim()));
+  };
+
+  const jsxInputFormat = (date: Date): string => {
+    const year = date.getFullYear();
+    const month = `${date.getMonth() + 1}`.padStart(2, "0");
+    const day = `${date.getDate()}`.padStart(2, "0");
+    const hours = `${date.getHours()}`.padStart(2, "0");
+    const minutes = `${date.getMinutes()}`.padStart(2, "0");
+    return `${year}-${month}-${day}T${hours}:${minutes}`;
```
Comment (cloudyyoung) [inline client/src/components/EditAssessment/EditAssessment.tsx:42]: Can use `toISOString()` instead.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString

**Review by cloudyyoung — APPROVED** [review]
Approved with a comment.

File: client/src/components/EditAssessment/EditAssessment.tsx (hunk 1 above)
Comment (tim-macphail) [reply client/src/components/EditAssessment/EditAssessment.tsx:42]: I did that at first, but it would display in UTC time instead of local

**Pull request merged**